src/inputs/TextFieldBase.tsx

```diff
@@ -79,26 +80,47 @@ export function TextFieldBase(props: TextFieldBaseProps) {
   return (
     <div css={Css.df.flexColumn.w100.maxw(px(550)).$} {...groupProps}>
       {label && !hideLabel && <Label labelProps={labelProps} label={label} suffix={labelSuffix} {...tid.label} />}
-      <ElementType
```

Comment:
This diff looks big, but most of this just got indented, recommend the "ignore whitespace" view.